host/src/types/routes.ts

```diff
@@ -8,12 +8,13 @@ export const AuthRoutes = {
 
 export const DashboardRoutes = {
   home: "/dashboard",
-  shortlets: "/shortlet",
+  shortlets: "/shortlets",
   shortletDetail: "/shortlet/:id",
   editShortlet: "/shortlet/edit/:id",
```

Comment:
```suggestion
  editShortlet: "/shortlets/edit/:id",
```

Reply:
I would suggest

```suggestion
  editShortlet: "/shortlet/:id/edit",
```

since it matches to flow of the URI